package.json

```diff
@@ -30,21 +30,41 @@
     "build": "babel ./src --out-dir ./dist --copy-files",
     "build:clean": "rimraf ./build",
     "build:watch": "babel ./src --out-dir ./dist --watch",
+    "git-add": "git add",
     "lint": "eslint --fix --max-warnings 0 -- .",
+    "lint-staged": "lint-staged",
     "migrate-files-to-mongo": "babel-node ./src/scripts/migrateFilesToDatabase mongo",
     "migrate-files-to-nedb": "babel-node ./src/scripts/migrateFilesToDatabase nedb",
     "prebuild": "npm run build:clean",
-    "start:warn": "babel-node ./src/main.js --trace-warnings",
+    "prettify": "prettier --single-quote --trailing-comma all --write",
     "start": "nodemon --exec babel-node ./src/main.js --watch src --watch ../spark-protocol/dist --ignore data",
     "start:debug": "nodemon --exec babel-node ./src/main.js --watch src --watch ../spark-protocol/src --ignore data",
     "start:prod": "node ./dist/main.js",
+    "start:warn": "babel-node ./src/main.js --trace-warnings",
     "test": "ava --serial --no-cache",
     "test:watch": "ava --watch --serial",
     "update-firmware": "node ./node_modules/spark-protocol/dist/scripts/update-firmware-binaries",
     "watch": "babel ./src --out-dir ./dist --watch"
   },
+  "lint-staged": {
```

Comment:
I wish I could do this nicer, without duplication, but lint-staged has very limited blob patterns. We can change it after they close the issues: https://github.com/okonet/lint-staged/issues/201 and https://github.com/okonet/lint-staged/pull/188.
Without lint-staged seems there is no simple way to add linted and prettified files back to commit, they keep modified and wait for next commit.